packages/interactive-messages/src/adapter.ts

```diff
@@ -640,4 +756,25 @@ type ActionHandler = (payload: any, respond: Respond) => any | Promise<any> | un
  */
 type OptionsHandler = (payload: any) => any | Promise<any> | undefined;
 
-type Callback = ActionHandler | OptionsHandler;
+/**
+ * A handler function for view submission requests.
+ *
+ * TODO: describe the payload and return values more specifically?
+ */
+type ViewSubmissionHandler = (payload: any) => any | Promise<any> | undefined;
```

Comment:
The types for these are in Bolt if you want them - https://github.com/slackapi/bolt/blob/master/src/types/view/index.ts

Reply:
Great idea!

Reply:
On second thought - the options are to depend on `@slack/bolt` to get these types, to duplicate those types in `@slack/interactive-messages`, or to move those types into `@slack/types` (and then refactor Bolt and this package to grab those types from that package).

I think the ideal solution is the last one, but I don't think its wise to increase scope of this feature to get that done. In the meantime, I think we can proceed with this approach and revisit the refactoring of all payload types into `@slack/types` at a later time. Many other packages might also benefit from this.